salt/cni/kube-flannel.yaml.jinja

```diff
@@ -103,8 +103,21 @@ spec:
           - "--kube-subnet-mgr"
           - "--v={{ pillar['flannel']['log_level'] }}"
           - "--iface=$(POD_IP)"
+          - "--healthz-ip=$(POD_IP)"
+          - "--healthz-port=8081"
```

Comment:
Nitpick: we might want to pillar this port.

Reply:
Fixed